TheJumpCloud/jumpcloud-ADMU
# Sa 2907 bind as admin

## Issues
* [SA-2907](https://jumpcloud.atlassian.net/browse/SA-2907) - Bind as administrator

## What does this solve?

New parameter to allow users to be bound as sudo enabled (or administrator)

## Is there anything particularly tricky?

## How should this be tested?

Unit & Functional tests were written to test the command line functionality of this new parameter set.

To manually test the form parameter set:

- Migrate a user with "Autobind JC User" Selected  | The user should be bound as a standard user
- Migrate a user with "Autobind JC User" & "Bind As Admin" Selected | The user should be bound as an Administrator

## Screenshots

![Screen Shot 2022-11-16 at 2 45 12 PM](https://user-images.githubusercontent.com/54448601/202301646-ba088214-3211-4845-8982-89117e45504d.png)
